src/8800.c

```diff
@@ -45,7 +45,7 @@ void update_cameras(void) {
                     break;
             }
 
-            guLookAtReflectF(cam->viewMtxPlayer, &gDisplayContext->lookAt[0], cam->lookAt_eye.x, cam->lookAt_eye.y, cam->lookAt_eye.z, cam->lookAt_obj.x, cam->lookAt_obj.y, cam->lookAt_obj.z, 0, 1.0f, 0);
+            guLookAtReflectF(cam->viewMtxPlayer, &gDisplayContext->lookAt, cam->lookAt_eye.x, cam->lookAt_eye.y, cam->lookAt_eye.z, cam->lookAt_obj.x, cam->lookAt_obj.y, cam->lookAt_obj.z, 0, 1.0f, 0);
```

Comment:
Should probably just be `gDisplayContext->lookAt` not `&gDisplayContext->lookAt`

Reply:
That would pass the lookAt by value, which we don't want